src/pages/UserSettings.vue

```diff
@@ -108,17 +100,6 @@ export default {
     eventSelectedImage() {
       this.$refs.fileImageProfile.click();
     },
-    uploadImage(event) {
```

Comment:
I think this might be useful
the idea is to save the images locally in order not to hit the db upon every single refresh this means that our server will be hit a lot lot less